controllers/provider-aws/pkg/apis/config/types.go

```diff
@@ -46,3 +51,20 @@ type RegionAMIMapping struct {
 	// AMI is the AMI for the machine image.
 	AMI string
 }
+
+// ETCD is an etcd configuration.
+type ETCD struct {
+	// StorageClassName is the name of the storage class used in etcd-main volume claims.
+	// +optional
```

Comment:
`// +optional` tags are not needed in the internal version

Reply:
Removed.